components/form/stories/form.stories.js

```diff
@@ -80,18 +79,19 @@ export default {
 				content: [
 					(passthroughs, context) => Fieldgroup({
 						layout: "horizontal",
+						inputType: "checkbox",
```

Comment:
The changes in `form` were inspired by a few PRs: 

- [fieldlabel & form docs migration](https://github.com/adobe/spectrum-css/pull/3165) defined these inputs as checkboxes
- [[field group regressions](https://github.com/adobe/spectrum-css/pull/3227)] added the `inputType` argument and changed the format of the `items` array a bit. 

Combined, we can address [the regressions seen in form](https://opensource.adobe.com/spectrum-css/?path=/story/components-form--default) (where the the "Interest" inputs have no labels)

Reply:
Beautiful, love these updates!